apis/pkg/meta/configuration_types.go

```diff
@@ -0,0 +1,40 @@
+/*
+Copyright 2020 The Crossplane Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package meta
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// ConfigurationSpec specifies the configuration of a Configuration.
+type ConfigurationSpec struct {
+	MetaSpec `json:",inline"`
+}
+
+// +kubebuilder:object:root=true
+// +kubebuilder:skipversion
```

Comment:
This annotation marks this version as internal - we don't include it in our generated CRD 'API docs'.